### PR TITLE
Adding support for FutureAccessList urn.

### PR DESCRIPTION
**Pull request by phongcao**
This PR handles a special urn for [FutureAccessList](https://docs.microsoft.com/en-us/uwp/api/windows.storage.accesscache.storageapplicationpermissions#Windows_Storage_AccessCache_StorageApplicationPermissions_FutureAccessList).
Uri format: 

> urn:future-access-list:{GUID}

### CODE REVIEW

File: ImagePipeline/ImagePipeline/Producers/FutureAccessListFetchProducer.cs [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        {
+        }
+
+        /// <summary>
+        /// Gets the encoded image.
+        /// </summary>
+        protected override Task<EncodedImage> GetEncodedImage(ImageRequest imageRequest)
+        {
+            string originalString = imageRequest.SourceUri.OriginalString;
+            string token = originalString.Substring(originalString.LastIndexOf(":") + 1);
```
Comment (antonkh) [inline ImagePipeline/ImagePipeline/Producers/FutureAccessListFetchProducer.cs]: This needs to be URL-decoded: `urn:future-access-list:%7D123-455-789%7B`

**Comment by antonkh** [comment]
As to why this is needed. In Skype the user can drag & drop an image file to send it in a conversation. Due to limitations of the UWP file model, the file needs to be put to FutureAccessList, so from that point Skype deals with a FAL token (which is just a GUID). Later it needs to get size of the dropped image, so it tells RN windows to load the image and RNW in turn uses this library to load images. This is why this library needs to be aware of FutureAccessList.

**Comment by phongcao** [comment]
Thanks for the context. It totally makes sense.

File: ImagePipeline/ImagePipeline/Producers/FutureAccessListFetchProducer.cs (hunk 1 above)
Comment (phongcao) [reply ImagePipeline/ImagePipeline/Producers/FutureAccessListFetchProducer.cs]: Fixed.

**Pull request merged**